lib/plugsnag.ex

```diff
@@ -32,7 +32,8 @@ defmodule Plugsnag do
       end
 
       defp redact(params) do
-        filtered = redact_config.fields
+        fields = redact_config[:fields] || ~w(password)
```

Comment:
maybe we should concat the defaults with the config settings instead of one over the other